tools/nav.txt

```diff
@@ -38,15 +38,15 @@
 +models.html@Models
```

Comment:
thanks, I had ping'd about it in irc the other day and it didn't seem like it was part of anything.